### ISSUE TITLE
[BUG] Tooltips no longer disappear, when opacity: 1

### ISSUE THREAD

**Issue by ansgarsteinkamp** [ReactTooltip/react-tooltip#1112]
Version: 5.22.0

Description: In order to be able to style the tooltips with Tailwind CSS, I use a class "classNamesTooltip", in which I set "opacity: 1". Since version 5.22.0, this results in the tooltips no longer disappearing.

![A](https://github.com/ReactTooltip/react-tooltip/assets/75034197/6df5f586-5031-4b96-a4fc-b7a06849bf31)

![B](https://github.com/ReactTooltip/react-tooltip/assets/75034197/a00d6668-fdbc-4451-aa03-8233b9f705ff)

![C](https://github.com/ReactTooltip/react-tooltip/assets/75034197/ea199898-d502-40a2-8999-26905a0f545e)

**Comment by gabrieljablonski** [comment]
Setting the opacity through CSS is not supported. Use the `opacity` prop on the tooltip instead.

```tsx
<Tooltip opacity={1} />
```

**Comment by EmanuelU** [comment]
I had the same issue, thanks!

**Comment by anch09** [comment]
I got the same bug, with the latest version at the date today which is 5.22.0 the tooltips are not dissappearing when mouserleave event is triggered.

**Comment by gabrieljablonski** [comment]
@anch09, as mentioned above, this is not a bug. Have you tried using the `opacity` prop?

#1113 will add this more clearly to the documentation.

**Comment by anch09** [comment]
@gabrieljablonski so it's intended to work as it from the version 5.22.0? because for me it's way better how it used to be before

**Comment by gabrieljablonski** [comment]
@anch09 yes, the previous behavior was not intended. If setting the opacity through CSS is better for you, you can also try overriding `--rt-opacity` like this

```css
:root {
  --rt-opacity: 1;
}
```

**Comment by targumon** [comment]
> Setting the opacity through CSS is not supported, even before 5.22.0 it wasn't supposed to work.

Wasn't aware of that. Somehow it DID work for me.
Anyway, I switched to use the `opacity={1}` prop.

👍 Works fine in a real browser - thanks! 
👎 But... throws in tests. I'm using react-test-library & jest and get `CSS.supports is not a function` errors.

For now I just mocked the function.
In file `setupTestsAfterEnv.js` I added:
```js
global.CSS.supports ||= () => true
```
(can probably also do it in `setupTests.js` - I don't remember the difference. They're both referred from `jest.config.js` in properties `setupFilesAfterEnv` & `setupFiles` respectively)


However, I think this line should be modified:
https://github.com/ReactTooltip/react-tooltip/blob/41e2ae5ed84a66d05f514a3a20f3fa31ae8e43a6/src/components/TooltipController/TooltipController.tsx#L285
to include a check:

```js
if (opacity && 'supports' in CSS && !CSS.supports('opacity', `${opacity}`)) {
```

(similar to what these guys did here: https://github.com/wojtekmaj/react-fit/pull/3/files )

LMK if you want me to open a PR and thanks again! 🙏
